open-metadata-implementation/user-interfaces/access-services-user-interface/pom.xml

```diff
@@ -68,21 +68,161 @@
+                                </goals>
+                                <configuration>
+                                    <executable>${basedir}/tools/node/node</executable>
+                                    <arguments>
+                                        <argument>${basedir}/tools/node_modules/polymer-cli/bin/polymer.js</argument>
+                                        <argument>build</argument>
+                                    </arguments>
+                                    <workingDirectory>${basedir}/src/main/static</workingDirectory>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-resources-plugin</artifactId>
+                        <version>3.1.0</version>
+                        <executions>
+                            <execution>
+                                <id>copy-resources</id>
+                                <phase>process-resources</phase>
+                                <goals>
+                                    <goal>copy-resources</goal>
+                                </goals>
+                                <configuration>
+                                    <outputDirectory>${basedir}/target</outputDirectory>
+                                    <includeEmptyDirs>true</includeEmptyDirs>
+                                    <resources>
+                                        <resource>
+                                            <targetPath>${basedir}/src/main/resources/public</targetPath>
+                                            <directory>${basedir}/src/main/static/build/es6-bundled</directory>
+                                            <filtering>false</filtering>
+                                        </resource>
+                                    </resources>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.springframework.boot</groupId>
+                        <artifactId>spring-boot-maven-plugin</artifactId>
+                        <version>${spring-boot.version}</version>
+                        <executions>
+                            <execution>
+                                <goals>
+                                    <goal>repackage</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.apache.rat</groupId>
+                        <artifactId>apache-rat-plugin</artifactId>
+                        <configuration>
+                            <excludes>
```

Comment:
Two thoughts
a) Are any of these generic enough that we could move to the top level rat configuration -- having them in one place makes them easier to manage?
b) Some items in rat ignore are candidates for git ignore too - if the build is run 'in situ' - could you consider adding appropriate entries there too?
(subject to constraints of scoping within profiles)

Reply:
On b) in particular - this is still an issue.

Since it could lead to files being erroneously checked in with 'git add .' I think it's important to fix before merge.

As an example on my mac I ended up with:

15:20 $ git status
On branch bogdan-sava-ui
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	open-metadata-implementation/user-interfaces/access-services-user-interface/src/main/resources/public/images/
	open-metadata-implementation/user-interfaces/access-services-user-interface/src/main/resources/public/index.html
	open-metadata-implementation/user-interfaces/access-services-user-interface/src/main/resources/public/manifest.json
	open-metadata-implementation/user-interfaces/access-services-user-interface/src/main/resources/public/node_modules/
	open-metadata-implementation/user-interfaces/access-services-user-interface/src/main/resources/public/service-worker.js
	open-metadata-implementation/user-interfaces/access-services-user-interface/src/main/resources/public/src/
	open-metadata-implementation/user-interfaces/access-services-user-interface/tools/


The .gitignore needs updating.

The RAT specs aren't critical - in fact it may be it's a good idea to keep them local - so happy to skip on that one.